### ISSUE TITLE
the experiment result 

### ISSUE THREAD

**Issue by yuxinyan** [LisaAnne/LocalizingMoments#8]
Hi,Lisa. I have tested your released model and the model trained with your training code, but neither these models can achieve the accuracy in your paper. Could you give me some help?

**Comment by LisaAnne** [comment]
What are the accuracy you are getting?  Is it off by half a point or so, or off by a lot?

**Comment by yuxinyan** [comment]
I retrained model in past few days. Below is the result.





| val  | rank@1 | rank@5 | iou | 
| ------------- | ------------- | ------------- | ------------- | 
| Rgb(released)  | 0.244258 | 0.754067 | 0.373925 |
| Flow(released)  | 0.262679 | 0.783971| 0.401582 |
| Fusion(released) | 0.276555 | 0.796172 | 0.419102 |
| Rgb(trained) |  0.236603 | 0.766268 | 0.361992 |
| Flow(trained) |  0.265311 | 0.784450 | 0.400611 |
| Fusion(trained) |  0.272010 |  0.793541 | 0.408446 |
| Fusion(paper) | 0.2757 | 0.7969 | 0.4170 |

| test | rank@1 | rank@5 | iou | 
| ------------- | ------------- | ------------- | ------------- | 
| Rgb(released)  | 0.231286| 0.733648|  0.354971 |
| Flow(released)  | 0.258393| 0.754041| 0.389482 |
| Fusion(released) | 0.270828| 0.785377| 0.405315 |
| Rgb(trained) |   0.224322|  0.733151|  0.337629|
| Flow(trained) |  0.258393| 0.761502| 0.381673|
| Fusion(trained) |  0.274310|  0.776921| 0.399055|
| Fusion(paper) | 0.2810 | 0.7821 | 0.4108 |

I tested your released model and my own model trained by your code.
Released model and trained model show good accuracy on val set. Though the accuracy of trained model (fusion) is off by a point on iou. 
On test set, neither these two models (fusion) can reach the accuracy posted in paper. It is off by almost a point on rank@1.

**Comment by LisaAnne** [comment]
Interesting.  I will look into this and get back to you; these models were retrained by me on a different machine, so that could be part of the issue.  Additionally, I played around with extracting RGB features in different ways (mainly frame rate) post deadline, and validated on the val set.  It is possible that the parameters I found on the val set don't do as well on the test set.  Apologies for the confusion!  It might be worth seeing how much values fluctuate with different random initializations.
